app/logger_config.py

```
@@ -0,0 +1,14 @@
+import logging
+import sys
+
+
+def setup_logger(name: str) -> logging.Logger:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)